bHY6yh-de4661/cod-yyyiou_bHY6yh-de4661.xml

```diff
@@ -0,0 +1,120 @@
+              <lb ed="#C"/>in nomine meo nec pollues nomen domini dei
+              <lb ed="#C"/>tui</quote>
+            </p>
+            <p xml:id="bHY6yh-de4661-d1e133">
+              ¶ <ref xml:id="bHY6yh-de4661-Rd1e132"> Yesus filius sydrach in ecclesiastico capiutlo 33</ref> <quote xml:id="bHY6yh-de4661-Qd1e137">Iura<lb ed="#C" break="no"/>tio 
+              non assuescat os tuum multi enim casus
+              <lb ed="#C"/>in illa Vir multum iurans implebitur
+              <lb ed="#C"/>iniquitate</quote>
+            </p>
+            <p xml:id="bHY6yh-de4661-d1e148">
+              ¶ <ref xml:id="bHY6yh-de4661-Rd1e149">Mattheus in euangelio capitulo V</ref> <quote xml:id="bHY6yh-de4661-Qd1e152">Noli
+              <lb ed="#C"/>iurare omnino neque per caelum quia tronus dei
+              <lb ed="#C"/>est neque per terram quia scabellum pedum
+              <lb ed="#C"/>eius est neque per Ierosolimam quia ciui<lb ed="#C" break="no"/>tas 
+              magni regis est neque per caput tuum
+              <lb ed="#C"/>iurabis quia non potes capillum unum
+              <lb ed="#C"/>facere aut nigrum Sit autem sermo uester est est
+              <lb ed="#C"/>non non quod autem ampluis est amalo
+              <lb ed="#C"/>est</quote>
+            </p>
+            <p xml:id="bHY6yh-de4661-d1e171">
+              ¶ <ref xml:id="bHY6yh-de4661-Rd1e174">Iacobus epistola I capitulo V et fidei</ref> <quote xml:id="bHY6yh-de4661-Qd1e175">Nolite iurare
+              <lb ed="#C"/>neque per caelum neque per terram neque per aliquid aliud
+              <lb ed="#C"/>Iuramentum sit autem autem sermo uester Est est non
+              <lb ed="#C"/>non</quote>
+            </p>
+            <p xml:id="bHY6yh-de4661-d1e185">
+              ¶ <ref xml:id="bHY6yh-de4661-Rd1e190">Paulus ad Ebreos capitulo VI</ref> <quote xml:id="bHY6yh-de4661-Qd1e189">per maiorem sui
+              <lb ed="#C"/>iurant et omnis controuersie eorum finis
+              <lb ed="#C"/>ad confirmationem est iuramentum</quote>
+            </p>
+            <p xml:id="bHY6yh-de4661-d1e199">
+              ¶ <ref xml:id="bHY6yh-de4661-Rd1e206">Cato
+              <lb ed="#C"/>capitulo I Iusiurandum serua</ref>
+            </p>
+            <p xml:id="bHY6yh-de4661-d1e207">
+              ¶ <ref xml:id="bHY6yh-de4661-Rd1e215">ysidorus soli<lb ed="#C" break="no"/>loquiorum 
+              libro 2 capitulo 35</ref> <quote xml:id="bHY6yh-de4661-Qd1e211">Assiduitas perui<lb ed="#C" break="no"/>rii 
+              consuetudinem facit Vsus iurandi
+              <lb ed="#C"/>ad peruirium ducit sit in ore tuo est sit in
+              <lb ed="#C"/>ore tuo non est veritas iuramento non in<lb ed="#C" break="no"/>diget </quote>
+            </p>
+            <p xml:id="bHY6yh-de4661-d1e223"> 
+              <quote xml:id="bHY6yh-de4661-Qd1e225">Firma sit sacramenti fides
+              <pb ed="#C" n="7-v"/>
+              <cb ed="#C" n="a"/>
+               </quote>
+            </p>
```

Comment:
If it ends at the bottom of column b, then the page break and column break declarations should go at the beginning  of the next file.
```suggestion
            <p xml:id="bHY6yh-de4661-d1e223"> 
              <quote xml:id="bHY6yh-de4661-Qd1e225">Firma sit sacramenti fides</quote>
            </p>
```